modules/account-lib/src/utils/helpers.ts

```diff
@@ -0,0 +1,61 @@
+import { randomBytes } from 'crypto';
+
+const _0n = BigInt(0);
+const _1n = BigInt(1);
+/**
+ * Modulo operator
+ * @description In javascript % actually gives you the reminder not modulo.
+ * @param {bigint} num
+ * @param {bigint} mod
+ * @returns {bigint}
+ */
+export function modulo(num: bigint, mod: bigint): bigint {
+  const result = num % mod;
+  // modulo operation should always return positive values for curve operations
+  return result >= _0n ? result : mod + result;
+}
+
+/**
```

Comment:
We already pull noble-secp as a dependency. In noble-secp master, I exposed this function. I have a [branch](https://github.com/brandonblack/noble-secp256k1/tree/bitgo) that includes build products in git so you can depend on a git version until Paul cuts a new release.

Reply:
I have updated it